core/spec.md

```diff
@@ -945,6 +949,35 @@ of the existing entity. Then the existing entity would be deleted.
 - Examples:
   - `2030-12-19T06:00:00Z`
 
+#### modelversion
```

Comment:
this section and the next one should be moved under: https://github.com/xregistry/spec/blob/dfde83550d8662beca8e89766e4be4e6d81d7816/core/spec.md#registry-model